.github/stale.yml

```diff
@@ -0,0 +1,48 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 90
+
+# Number of days of inactivity before a stale Issue or Pull Request is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - WIP
+  - work in progress
+  - important
+  - feature-request
```

Comment:
I think "feature" is the label we use here?

Maybe add "bug" too?

Reply:
You are correct.

Will fix it and add "bug" as well - agreed.

Reply:
@LindsayHill Updated the config.